CHANGELOG.md

```diff
@@ -0,0 +1,45 @@
+# Changelog
+
+## 0.0.53
+
+### Added
+- Resource-oriented design for models, shields, memory banks, datasets and eval tasks
+- Persistence for registered objects with distribution
+- Ability to persist memory banks created for FAISS
+- PostgreSQL KVStore implementation
+- Environment variable placeholder support in run.yaml files
+- Provider deprecation support
+- Comprehensive Zero-to-Hero notebooks and quickstart guides
+- Colab notebook integration for Llama Stack lesson
+- Remote::vllm provider with vision model support
+- Support for quantized models in Ollama
+- Vision models support for Together, Fireworks, Meta-Reference, and Ollama
+- Bedrock distribution with safety shields support
+
+### Changed
+- Split safety into distinct providers (llama-guard, prompt-guard, code-scanner)
+- Improved distribution registry with SQLite default storage
+- Enhanced test infrastructure with composable fixtures
```

Comment:
is this really what happened in 0.0.52 -> 0.0.53?

Reply:
i dont think so. removing this.